# request
---
# response
float32[] angles
float32[] velocities
float32[] torques
float32[] endpoint_geom
float32[] endpoint_vel